src/Chassis.Fairing.WebApi/IChassisFairingWebApiMarker.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Chassis.Fairing.WebApi
+{
+    public interface IChassisFairingWebApiMarker

```

Comment:
you specify assemblies in the start up of chassis currently. so each assembly has a marker interface at the moment to facilitate scanning. I eventually want to move to an Assembly Attribute model.